cmd/agentctl/commands/log.go

```diff
@@ -0,0 +1,143 @@
+//  Copyright (c) 2019 Cisco and/or its affiliates.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at:
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package commands
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ligato/cn-infra/logging"
+	"github.com/ligato/vpp-agent/cmd/agentctl/utils"
+)
+
+func NewLogCommand(cli *AgentCli) *cobra.Command {
```

Comment:
Command `log set` could return an error when attempting to set the log level to a non-existing logger, or when using invalid log level.